frontend/src/app/home/containers/Home.js

```diff
@@ -121,11 +121,66 @@ export default class Home extends React.Component {
           });
           return null;
         });
+
+
+  }
+
+  sellStock(stock, quantity, price){
```

Comment:
This function doesn't do anything and can go away.

Reply:
Will remove